package.json

```diff
@@ -23,6 +23,7 @@
     "husky": "^8.0.3",
     "lint-staged": "^13.1.2",
     "rollup": "^3.10.1",
+    "serve": "^14.2.0",
```

Comment:
oh nice!
I didn't think it can be possible with only one additional dependency.
and I would like this regardless of this PR anyway since non-es-module three.js will be deprecated.